purescript.cabal

```diff
@@ -962,7 +964,7 @@ executable purs
 
 test-suite tests
     build-depends:
-          aeson >=1.0 && <1.2
+          aeson >=1.0 && <1.1
```

Comment:
Hmm, looks like that PR that went in changed this instead of the package.yaml. Maybe this should be removed from and gitignored in the repo now, if we're relying on hpack?